meson_options.txt

```diff
@@ -0,0 +1,11 @@
+# Feature options
+option('enable-tests', type : 'boolean', value : true, description : 'Enable tests')
+option('enable-check', type : 'boolean', value : true, description : 'Enable check tests, need the enable-tests flag set')
+option('enable-graphviz', type : 'boolean', value : false, description : 'Enable graphviz usage')
+option('enable-gtk-doc', type : 'boolean', value : true, description : 'Use gtk-doc to build documentation')
```

Comment:
Same here. I think we should think in the functionality rather than the technology. We can use `enable-docs`